enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -126,6 +127,9 @@ func (c *podmanCommandContainer) getPodmanRunArgs(workDir string) []string {
 		}
 		args = append(args, "--device="+strings.Join(deviceSpecs, ":"))
 	}
+	for _, mount := range c.options.Mounts {
+		args = append(args, "--mount="+mount)
+	}
```

Comment:
Podman supports volumes too, so I think we'd want to use the same flag here to be consistent with Docker.

```suggestion
	for _, volume := range c.options.Volumes {
		args = append(args, "--volume="+volume)
	}
```